upkeep.py

```diff
@@ -172,7 +172,8 @@ def get_upcoming_workdays(workdays_ahead: int = 2) -> typing.Iterator[datetime.d
     # Get n most recent issues (open or closed), where n = 10 + --workdays-ahead
     # to help ensure the most recent existing e-scrum issue is included even when other
     # non e-scrum issues exist
-    issues = repo.get_issues(state='all', sort='number', direction='desc')[:10 + args.workdays_ahead]
+    issues = repo.get_issues(state='all', sort='number', direction='desc')
+    issues = issues[:min(10 + args.workdays_ahead, issues.totalCount)]
```

Comment:
I am surprised we got an index out of range error. Seems like a pygithub bug with their `github.PaginatedList.PaginatedList`.

Would expect this to behave like a Python list.

Anyways, thanks for the fix!